src/leap/mail/imap/mailbox.py

```diff
@@ -514,13 +514,14 @@ def _get_imap_msg(messages):
                 d_imapmsg = []
                 for msg in messages:
                     d_imapmsg.append(getimapmsg(msg))
-                return defer.gatherResults(d_imapmsg)
+                return defer.gatherResults(d_imapmsg, consumeErrors=True)
 
             def _zip_msgid(imap_messages):
                 zipped = zip(
                     list(msg_range), imap_messages)
                 return (item for item in zipped)
 
+            # XXX not called??
```

Comment:
Ok. I'm wondering if we ever have time to look at our `XXX` comments, or we are just piling them up. But fine for me, let's have a look to that in another time.